drivers/i3c/i3c_common.c

```diff
@@ -216,6 +216,24 @@ struct i3c_device_desc *i3c_dev_list_i3c_addr_find(const struct device *dev,
 	return ret;
 }
 
+struct i3c_device_desc *i3c_dev_list_i3c_static_addr_find(const struct device *dev,
+						   uint8_t addr)
```

Comment:
```suggestion
struct i3c_device_desc *i3c_dev_list_i3c_static_addr_find(const struct device *dev,
							  uint8_t addr)
```

Reply:
double post?

Reply:
No, both header and implementation.

Reply:
Comment still pending here.

Reply:
as I was making changes per @dcpleung request, I just went ahead and did this